docs/tutorial/importing_targets.rst

```diff
@@ -0,0 +1,29 @@
+.. # Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. # other BLT Project Developers. See the top-level LICENSE file for details
+.. # 
+.. # SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _ImportingTargets:
+
+Importing Targets
+=================
+
+One key goal for BLT is to simplify the use of external dependencies and libraries when building
+your project. To accomplish this, BLT provides a ``DEPENDS_ON`` option for the ``blt_add_library()``
+and ``blt_add_executable()`` macros that supports both your own projects CMake targets 
+and imported targets. We have logically broken this topic into two groups: :ref:`CommonHPCDependencies`
```

Comment:
Suggestion: This might be easier to parse as a bullet list with two entries.

E.g.
```
We have logically broken this topic into two groups: 

* :ref:`CommonHPCDependencies`, such as MPI, ...
* :ref:`ThirdPartyLibraries` are imported into ...

The next two pages describe these in detail.

<TOC>

```